MapboxCoreNavigation/MBFixLocation.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+import MapboxNavigationNative
+import CoreLocation
+
+extension MBFixLocation {
+    
+    func asCLLocation() -> CLLocation {
+        let coordinate = CLLocationCoordinate2D(latitude: 0, longitude: 0)
+        let altitude: CLLocationDistance = self.altitude?.doubleValue ?? 0
+        let horizontalAccuracy: CLLocationAccuracy = self.accuracyHorizontal?.doubleValue ?? 0
+        let course: CLLocationDirection = self.bearing?.doubleValue ?? 0
+        let speed: CLLocationSpeed = self.speed?.doubleValue ?? 0
+        let timestamp: TimeInterval = self.time?.doubleValue ?? Date().timeIntervalSince1970
```

Comment:
Nit, self is not needed here i think x 7.

Reply:
It's not needed here or on any of the lines where the local name differs from the instance variable. e.g. `let speed: CLLocationSpeed = self.speed`, `speed` shadows `speed` so `self` can't be inferred.